hathor/transaction/transaction_metadata.py

```diff
@@ -344,3 +353,8 @@ def clone(self) -> 'TransactionMetadata':
         """
         # XXX: using json serialization for simplicity, should it use pickle? manual fields? other alternative?
         return self.create_from_json(self.to_json())
+
+    def get_soft_height(self) -> int:
```

Comment:
Missing docstring. Please clarify why we need the fallback that returns `self.height`?

Reply:
I improved it a little.